bedrock/mozorg/templates/mozorg/developer/index.html

```diff
@@ -31,7 +31,13 @@
 {% endblock %}
 
 {% block content %}
-{% include 'mozorg/developer/includes/sub-nav.html' %}
+
+{% if LANG.startswith('en-') %}
+  {% include 'mozorg/developer/includes/dev-tech-sub-nav.html' %}
```

Comment:
I'm not entirely sure we need a second nav (afaik the developer page is not localized yet).